sdk/storage/Microsoft.Azure.WebJobs.Extensions.Storage.Scenario.Tests/tests/CacheableBlobsEndToEndTests.cs

```diff
@@ -112,15 +116,44 @@ public async Task BindToCacheableReadBlob_VerifyCacheHit()
         }
 
         [Test]
-        public void BindCacheableWriteBlob_VerifyPutToCache()
+        public async Task BindToCacheableWriteBlob_VerifyBlobStream()
         {
-            // TODO
+            // Invoke the function which will verify that the input object has a BlobStream bound to it
```

Comment:
what's the assertion here? lack of exception thrown?

Reply:
The assertion is inside the function `ByteArrayBindingWriteToCacheAsync` that will be called. So if that fails, then this throws.